.github/workflows/pre-release-continuous-test.yaml

```diff
@@ -9,15 +9,24 @@ on:
   push:
     branches:
       - pre-release-continuous-test
+      # remove this before landing PR
+      - rustielin/forge-multi-nightly
   schedule:
-    # Run hourly
-    - cron: "0 * * * *"
+    # Run every 3 hours
+    - cron: "0 */3 * * *"
 
 jobs:
-  run-forge:
+  # run two concurrent forge test jobs on the same cluster
+  # they must use different namespaces, or they will preempt each other
+  run-forge-0:
```

Comment:
@rustielin - Why limit this to two concurrent jobs? Is there a way we can run many jobs in parallel as long as we have capacity?

Reply:
yea we can run as many as we have capacity for. do you have an idea for how many you'd want? today we're lacking the test suites to run, but this is all configurable

Reply:
In my mind, I think we would run ~10 tests. I am okay with starting with 2 concurrent tests as long as we can increase this later.

Reply:
Yes, we can run them in sequence to start with and monitor how long they take. If it takes more than 6 hours, we might need to consider other optimizations.